### PR TITLE
Restore descriptions and display

### PR DESCRIPTION
**Pull request by Coperh**
Fixes: https://github.com/open-zaak/open-zaak/issues/1490

### CODE REVIEW

**Review by annashamray — CHANGES_REQUESTED** [review]
It looks quite neat, I like the approach

File: vng_api_common/choices.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        if choice not in descriptions:
+            raise ValueError(
+                f"Choice ({choice}, {text}) in {cls.__name__} is missing a description"
+            )
+    return cls
+
+
+class TextChoicesWithDescriptions(TextChoices):
+    @classmethod
+    def get_descriptions(cls):
```
[the remark this answers is not in the document]
Comment (Coperh) [reply vng_api_common/choices.py]: Done

File: vng_api_common/apps.py [hunk 2]
```diff
@@ -88,3 +90,8 @@ def set_charfield_error_messages():
             "min_length": _("The value has too few characters"),
         }
     )
+
+
+def ensure_text_choice_descriptions():
+    for cls in TextChoicesWithDescriptions.__subclasses__():
```
[the remark this answers is not in the document]
Comment (Coperh) [reply vng_api_common/apps.py]: Made it recursive, so it should run on subclasses.

File: vng_api_common/choices.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from django.db.models import TextChoices
+
+
+def ensure_description_exists(cls):
```
[the remark this answers is not in the document]
Comment (Coperh) [reply vng_api_common/choices.py]: Done

**Review by annashamray — APPROVED** [review]

**Pull request merged**